gradle/versions.gradle

```diff
@@ -15,7 +15,6 @@ ext {
     cucumberVersion = '1.2.5'
     hamcrestVersion = '1.3'
     javaxServletApiVersion = '3.1.0'
-    jacksonVersion = '2.9.2'
```

Comment:
The jackson isn't used anymore?

Reply:
It is but this variable is redundant and never used. jacksonCoreVersion is used instead.